internal/nodes/ingress_model_rel.go

```diff
@@ -215,6 +221,27 @@ func SvcToGateway(svcName string, namespace string, key string) ([]string, bool)
 	return allGateways, true
 }
 
+// PodToRoute fetches the list of impacted Routes from Pod update.
+// First fetch list of Services for the Pod.
+// Then get list of Routes for the services.
+func PodToRoute(podName string, namespace string, key string) ([]string, bool) {
```

Comment:
NodePortLocal isn't available with OpenShift currently, should we hold off from introducing this code till that happens?

Reply:
I wanted to keep it just to be compatible. But I guess we can add it later when Antrea is used in Openshift. Removed for now.